mountainduck/issues.md

```diff
@@ -87,6 +87,10 @@ The cache directory is located in `%LocalAppData%\Cyberduck\Cache` on Windows or
 :width: 600px
 ```
 
+## Insufficient disk space 
+
+If the available disk space on the mounted volume is below 100MB a soft quota notification will be displayed saying *Insufficient disk space*.
+
```

Comment:
Add screenshot of notification.

Reply:
Add notice, that quota information is only available for selected protocols. https://github.com/iterate-ch/docs/issues/80#issuecomment-1007485059